src/App.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import './App.css';
 import TodoForm from './components/TodoComponents/TodoForm';
 import TodoList from './components/TodoComponents/TodoList';
 
```

Comment:
remember we want the ability to toggle the style, we want something like the opposite of the current state, so if completed is true: apply css, if not, remove, we can probably achieve this by using !completed with creating a function to toggle it